components/setting_picture.tsx

```diff
@@ -53,7 +65,7 @@ export default class SettingPicture extends Component {
     }
 
     focusFirstElement() {
-        if (this.settingList.current) {
+        if (this.settingList && this.settingList.current) {
             this.settingList.current.focus();
         }
```

Comment:
We can use optional chaining here
```suggestion
        this.settingList.current?.focus();
```